ashpaper-bin/Cargo.toml

```diff
@@ -18,7 +18,7 @@ name = "ashpaper"
 path = "src/main.rs"
 
 [dependencies.ashpaper]
-version = "0.2.5"
+path = "../ashpaper"
```

Comment:
I've been trying to remember to toggle this back and forth when the version on github is ahead of the version on crates.io. It'd be cool to be able to just use one or the other based on a flag to cargo but idk how to do that atm...